src/tensors/planar.jl

```diff
@@ -219,7 +219,7 @@ function _add_modules(ex::Expr)
                         (ex.args[i] for i in argind)...)
     elseif ex.head == :call && ex.args[1] == :trace!
         @assert ex.args[4] == :(:N)
-        argind = [2,4,5,6,7,8,9,10]
+        argind = [2,3,4,5,6,7,8,9,10]
```

Comment:
Change this to `[2,3,5,6,7,8,9,10]`. The `@planar` macro will take out `conj` statements and replace them with adjoints, that's why there is `@assert ex.args[4] == :(:N)`.

This then affects the `planar_trace!` function below, it should not have the conjugation flag as an argument.

Reply:
Ah, I was wondering about whether this flag fill actually ever do anything, but I guess I should have read the line above. :)

I'll then remove the split between `planar_trace!` and `_planar_trace!`, because it's ugly and I only did it to deal with `AdjointTensorMap` vs `TensorMap`.